### ISSUE TITLE
Error executing view, "dcp_died"

### ISSUE THREAD

**Issue by sethrosetter** [couchbase/sync_gateway#2508]
### Sync Gateway version
1.4.2-341

### Operating system
CentOS 7

### Config file
```
{
        "adminInterface": "0.0.0.0:4985",
        "compressResponses": false,
        "databases": {
            "db": {
                "bucket": "data-bucket",
                "import_docs": "continuous",
                "password": "password",
                "server": "http://cbl.0:8091",
                "unsupported": {
                    "enable_extended_attributes": true
                },
                "users": {
                    "GUEST": {
                        "admin_channels": [
                            "*"
                        ],
                        "disabled": true
                    }
                }
            }
        },
        "interface": ":4984",
        "log": [
            "CRUD+",
            "Cache+",
            "HTTP+",
            "Changes+"
        ],
        "maxCouchbaseConnections": 16,
        "maxFileDescriptors": 90000,
        "maxIncomingConnections": 0,
        "slowServerCallWarningThreshold": 500
    }
```

### Log output
https://gist.github.com/sethrosetter/2c9c09d8878ada105582a5c9a51ff3f9

### Expected behavior
Doc should be deleted.

### Actual behavior
500 Internal Server Error on view execution

### Steps to reproduce
1. `http POST http://cbl.1:4985/db/ foo=bar`
2.  `http DELETE http://cbl.1:4984/db/a544e33b3fb3a1e3e64616c628b993c6 rev=1-cd809becc169215072fd567eebd8b8de`

**Comment by sethrosetter** [comment]
Using Couchbase Server `5.0.0-2759`

**Comment by adamcfraser** [comment]
This is supposed to be fixed on 5.0.0-2761.

https://issues.couchbase.com/browse/MB-21350?focusedCommentId=213136&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-213136

Please reopen if you see the 'dcp_died' error on 2761 or later.
